setup.cfg

```diff
@@ -77,11 +77,11 @@ exclude =
     build,
     dist,
 	nilearn/externals/tempita
---select = D,E,F,W,C90
+--select = D,E,F,W,C
 docstring-convention = numpy
 max-line-length = 79
-max_complexity = 43
-max_function_length = 150
+max_complexity = 34
+max_function_length = 255
```

Comment:
Note that max_function_length is not (yet?) checked in CI, this value would only trigger a flaek8 error if you ran flake8 locally with flake8-functions installed.

I updated this value so that once this PR is done, there should be no function that are longer than 255 lines (see the list here https://github.com/nilearn/nilearn/issues/3655).

This way if we want to implement function metric checks in CI, this new value will be our upper limit that gives us a green CI.

Reply:
Thanks for the explanations @Remi-Gau ! 🙏 
Great to see these massive functions getting chunked ! 👍